### PR TITLE
PIM-4873 : override last operations dashboard widget

### PR DESCRIPTION
**Pull request by jmleroux**
| Q                  | A
| ------------------ | ---
| Migration script?  |n
| Behats             |y
| Specs & Static     |y
| Changelog updated  |n

### CODE REVIEW

File: src/Pim/Bundle/DashboardBundle/Resources/public/js/widget-container.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    [
+        'backbone'
+    ],
+    function (Backbone) {
+        'use strict';
+
+        /**
+         * Widget container maintain a widget registry to use them on dashboard.
+         */
+        var WidgetContainer = Backbone.Model.extend({
```
Comment (sumbobyboys) [inline src/Pim/Bundle/DashboardBundle/Resources/public/js/widget-container.js]: please, don't override model for that.

File: src/Pim/Bundle/DashboardBundle/Resources/config/requirejs.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 config:
     paths:
+        pim/dashboard/widget-container:       pimdashboard/js/widget-container
```
Comment (grena) [inline src/Pim/Bundle/DashboardBundle/Resources/config/requirejs.yml]: I think we follow abcd order in those files

**Comment by grena** [comment]
Looks good to me, GTM if behat ok :)

**Comment by sumbobyboys** [comment]
GTM

**Pull request merged**